### PR TITLE
Fix '+' operator use with Paths

### PR DESCRIPTION
**Pull request by michalfabik**
Leftover from https://github.com/abrt/retrace-server/pull/310

Resolves:
https://github.com/abrt/retrace-server/issues/329
Signed-off-by: Michal Fabik <mfabik@redhat.com>

### CODE REVIEW

File: src/retrace/retrace.py [hunk 1]
```diff
@@ -1125,13 +1125,13 @@ def prepare_debuginfo(self, task, chroot=None, kernelver=None):
                 kernel_path = kernel_path + "."
             kernel_path = kernel_path + str(kernelver.flavour)
 
-        vmlinux_cache_path = debugdir_base + "usr/lib/debug/lib/modules" / kernel_path / "vmlinux"
+        vmlinux_cache_path = debugdir_base / "usr/lib/debug/lib/modules" / kernel_path / "vmlinux"
         if vmlinux_cache_path.is_file():
-            log_info("Found cached vmlinux at path: " + vmlinux_cache_path)
+            log_info("Found cached vmlinux at path: " + str(vmlinux_cache_path))
```
Comment (xsuchy) [inline src/retrace/retrace.py:1130]: I would slightly prefer:

`"Found cached vmlinux at path: {}".format(vmlinux_cache_path)

Many people find it more readable.
`

Comment (michalfabik) [reply src/retrace/retrace.py]: Thanks! It's actually safer because `.format()` works even if the arg is a `Path`, unlike `+`.

**Comment by ernestask** [comment]
Good to go or should I wait?

**Comment by michalfabik** [comment]
Good to go. \*fingers crossed\*

**Pull request merged**